### DISCUSSION TITLE
Cloudflare Rocket Loader no login screen

### DISCUSSION THREAD

**Discussion by itsMaxio** [nextcloud/all-in-one#1677] [Ideas] [answered]
Add information in readme about Cf [Rocket Loader](https://help.nextcloud.com/t/login-page-not-working-solved/149417/8). The problem was with cloudflare tunnel and Rocker Loader option in Speed section but after disabling this option the login box showed up :)
I think this should be in readme.

![image](https://user-images.githubusercontent.com/29357824/210834448-8903d906-2a47-41c3-a5ef-733ca389b554.png)

**Comment by szaimen** [comment]
Hi, can you point me to a specific part in the readme where we should add this in your opinion?

**Comment by itsMaxio** [comment]
In the [Reverse Proxy Documentation](https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md#cloudflare-argo-tunnel) in the Cloudflare section. Btw name argo tunnel is kinda outdated i think 
![image](https://user-images.githubusercontent.com/29357824/211159880-c1d226fb-76ff-4889-b0a9-fddcd5d8884a.png)

**Comment by szaimen** [comment, marked as answer]
Will be handled in https://github.com/nextcloud/all-in-one/pull/1696
